src/components/cc-zone/cc-zone.js

```diff
@@ -144,6 +144,7 @@ export class CcZone extends LitElement {
           --lh: 1.5em;
 
           display: flex;
+          align-items: center;
```

Comment:
**💣 chore:** Actually, to fix `cc-addon-linked-apps` alignment issue, I would not add this rule here but... (1/2)

```suggestion
```